other_procore_apis/bim_web_viewer.md

```diff
@@ -839,6 +796,540 @@ undefined
+
+Fires when the 2D Navigation overlay is used to navigate the model. Specifically, this fires after a location and direction have been chosen on the floor plan and the overlay is dismissed.
+
+---
+
+### drawingMiniClick
+
+<p class="heading-link-container"><a class="heading-link" href="#drawingminiclick"></a></p>
+
+Fires when the 2D Navigation mini map was clicked on.
+
+---
+
+### floorPlan
+
+<p class="heading-link-container"><a class="heading-link" href="#floorplan"></a></p>
+
+Fires when the 2D Navigation overlay is interacted with.
+
+#### Data Properties
+
+| Field Name                | Type        | Description                                                                                                 |
+| ------------------------- | ----------- | ----------------------------------------------------------------------------------------------------------- |
+| floorPlanNavigateAndClose | function    | Provides a callback function to navigate the camera to a position and close the overlay. Returns a promise. |
+| floorPlanPoint            | function    | Provides a callback function to calculate a point on the current floor plan. Returns a promise.             |
+| overlay                   | HTMLElement | The HTMLElement for the 2D Navigation overlay                                                               |
+| type                      | string      | The type of interaction and has the following values: `open`, `close`, and `update`                         |
+
+### floorPlanNavigateAndClose
+
+<p class="heading-link-container"><a class="heading-link" href="#floorplannavigateandclose"></a></p>
+
+Updates the camera position and direction in addition to dismissing the 2D floor plan overlay.
+
+#### Parameters
+
+| Field Name | Required | Type   | Description                |
+| ---------- | -------- | ------ | -------------------------- |
+| x          | true     | number | x coordinate in 3D space.  |
+| y          | true     | number | y coordinate in 3D space.  |
+| z          | true     | number | z coordinate in 3D space.  |
+| dx         | true     | number | x direction of the camera. |
+| dy         | true     | number | y direction of the camera. |
+
+### floorPlanPoint
+
+<p class="heading-link-container"><a class="heading-link" href="#floorplanpoint"></a></p>
+
+Returns the corresponding point on the 2D floor plan overlay from a point in 3D model space.
+
+#### Parameters
+
+| Field Name | Required | Type   | Description                 |
+| ---------- | -------- | ------ | --------------------------- |
+| x          | true     | number | x coordinate of the camera. |
+| y          | true     | number | y coordinate of the camera. |
+| z          | true     | number | z coordinate of the camera. |
+| dx         | true     | number | x direction of the camera.  |
+| dy         | true     | number | y direction of the camera.  |
+
+#### Example
```

Comment:
No change here. Children of `floorPlan` are:
- Data Properties
- `floorPlanNavigateAndClose`
- `floorPlanPoint`
- Example

Feels right to me.